float64 velocity_right
float64 velocity_left
---
bool success